O/OpenCC/build_tarballs.jl

```diff
@@ -0,0 +1,28 @@
+using BinaryBuilder
+
+name = "libopencc"
+version = v"1.1.3"
+sources = [
+  ArchiveSource("https://github.com/BYVoid/OpenCC/archive/refs/tags/ver.1.1.3.tar.gz", "99a9af883b304f11f3b0f6df30d9fb4161f15b848803f9ff9c65a96d59ce877f")
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/OpenCC-ver.1.1.3
+cmake -S. -Bbuild -DCMAKE_INSTALL_PREFIX:PATH=${prefix}
+cmake --build build --config Release --target install
```

Comment:
Please read the [documentation](https://docs.binarybuilder.org/stable/build_tips/#CMake-builds).  In particular, note that setting `CMAKE_TOOLCHAIN_FILE` is ***necessary*** to actually use the right compiler
```suggestion
mkdir build && cd build
cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    ..
make -j${nproc}
make install
```